doc/source/install/install.md

```diff
@@ -18,7 +18,7 @@ listed and explained in below table.
 | Convention | Usage |
 | ---------- | ----- |
 | Host Machines | Machines to be used for Openstack deployment. Openstack node controller, compute, storage and network node will be deployed on these machines. |
-| Configuration node | Machine running installation scripts, and hosting PXE, DHCP, TFTP services. |
+| Build Node | Machine running installation scripts, and hosting PXE, DHCP, TFTP services. |
```

Comment:
I would do Build Server, but let's have others comment as well.

Reply:
I agree with 'Build Server' as it isn't really a node.

Reply:
Changed all references to Build Node to Build Server